CryptoTracker/CryptoTracker/Scenes/Detail/DetailViewController.swift

```diff
@@ -6,14 +6,153 @@
 //
 
 import UIKit
+import Charts
 
 final class DetailViewController: UIViewController {
-    var uuid = ""
+
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet private weak var changeLabel: UILabel!
+    @IBOutlet private weak var iconImage: UIImageView!
+    @IBOutlet private weak var sympolLabel: UILabel!
+    @IBOutlet private weak var rankLabel: UILabel!
+    @IBOutlet private weak var isFavourite: UIButton!
+    @IBOutlet private weak var lineChartView: LineChartView!
+    @IBOutlet private weak var segment: UISegmentedControl!
+    @IBOutlet private weak var marketCap: UILabel!
+    @IBOutlet private weak var volumeTrading: UILabel!
+    @IBOutlet private weak var circulating: UILabel!
+    @IBOutlet private weak var bitcoinPrice: UILabel!
+    @IBOutlet private weak var allTimeHigh: UILabel!
+    @IBOutlet private weak var totalSupply: UILabel!
+
+    private var historyPrices = [History]()
+    private var coin: DetailCoin?
+    internal var uuid = ""
```

Comment:
@hanh-1287 ? Biến này em tương tác ở file khác à em ?

Reply:
vâng ạ

Reply:
@thuannv-2723 Vâng ạ

Reply:
@hanh-1287 . Không được phép thế em. Không đảm bảo được tính toàn vẹn của Controller này? 
Em đổi về private và viết 1 hàm thay đổi thuộc tính này. Sau đó call ở chỗ khác nhé.

Reply:
@thuannv-2723 Vâng ạ